test/controllers/images_controller_test.rb

```diff
@@ -24,4 +24,13 @@ class ImagesControllerTest < ActionDispatch::IntegrationTest
     end
     assert_response :redirect
   end
+
+  test 'should show images on homepage' do
+    Image.create(link: 'http://www.gstatic.com/tv/thumb/persons/80696/80696_v9_bb.jpg')
+
+    get images_url
+
+    assert_select 'img[src="http://www.gstatic.com/tv/thumb/persons/80696/80696_v9_bb.jpg"]'
+    assert_response :success
+  end
```

Comment:
The commit message says "show 20 images on the homepage". I would expect a test that verifies 20 images. Can you add one easily?